index.html

```diff
@@ -7,26 +7,32 @@
     <link href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" rel="stylesheet">
   </head>
   <body>
-    <header class="header">
-      <ul class="headerul">
-        <li>ABOUT US</li>
-        <li>RANGE</li>
-        <li>PROJECTS</li>
-        <li>OPINIONS</li>
-        <li>CONTACT</li>
-        <img src="img/united-kingdom.png" alt="その他">
-      </ul>
+    <section class="section1">
       <div class="clearfix">
```

Comment:
`<div class="clearfix">`はレイアウト上、意味を持たいない`div`になりなすので`<header class="header clearfix">`のように`header`に`clearfix`を追加してください。

Reply:
追加しました。

同様に
``<section class="aboutus">``の上の
``<div class="clearfix">``も修正したところ、
下のRangeが被ってしまい、色々試したけど直せませんでした。
以下要素はどこにあてるのが正しいのでしょうか。
``.clearfix::after {
  content: "";
    display: block;
    clear: both;
  }
``

Reply:
Rangeとかぶってしまう原因は今まで効いてなかったheightが反映されているからです。
下記heightを修正してください。
```suggestion
.aboutus {
  height: 598.3px;
  width: 1440px;
```

Reply:
できました！ありがとう。